frontend/src/modules/member/config/filters/segment/config.ts

```diff
@@ -0,0 +1,39 @@
+import { FilterConfigType } from '@/shared/modules/filters/types/FilterConfig';
+import {
+  MultiSelectFilterConfig,
+  MultiSelectFilterOptions,
+  MultiSelectFilterValue,
+} from '@/shared/modules/filters/types/filterTypes/MultiSelectFilterConfig';
+import { itemLabelRendererByType } from '@/shared/modules/filters/config/itemLabelRendererByType';
+
+const segment: MultiSelectFilterConfig = {
+  id: 'segment',
+  label: 'Segments',
+  iconClass: 'ri-apps-2-line',
+  inBody: true,
+  type: FilterConfigType.MULTISELECT,
```

Comment:
This will probably need to be 'custom' with a custom Component:
Multi Select won't match this component right?
![Screenshot 2023-05-29 at 11 23 54](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/be1e37f8-ec90-4143-a1d9-d3dd114b8f38)